### ISSUE TITLE
Plugin window says "Plugin Not Valid", then my audio pipe breaks

### ISSUE THREAD

**Issue by CJunit** [pierreguillot/Camomile#129]
Thank you for the new build script! I tried compiling Camomile a few months ago but got lost somehow. This time I was astonished that I got no errors.

The VST instrument and effect are recognized and load. In Waveform 9.2.1 and Bitwig demo 2.3.5 the plugins load and appear normal, but there is text in the middle that says "Plugin Not Valid." When I click on the flower a Pd console says:
camomile can't find the configuration file "home/user/.vst/Camomile/Camomile.txt"
camomile can't find the configuration file "home/user/.vst/Camomile/Camomile/Camomile.txt"
In Waveform and Bitwig then, my audio pops and there is shrill noise, as if Jack has lost its connection. In Carla, the window frame is movable, but the contents - the "Plugin Not Valid" and flower - are affixed to the upper-left of my monitor as if they are part of my desktop. That's all I know!
      
My system is KDE Neon, Plasma 5.13.3, frameworks 5.48.0, 4.16.12-rt5-avi1 kernel, current KXstudio installs and scripts, jackd2 2:1.9.12-1-xenial1, Nouveau graphics drivers 1:1.0.15-2-16.04.1. Hardware is a Dell XPS-9000, i7 920 @ 2.67GHz, 6GB RAM, Nvidia GTX260, Sonnet Allegro FW400 PCIe card connected to MOTU Traveller 

If there's any other version or configuration info I can provide that might elucidate why Camomile runs this way on this system, please do let me know what and I shall attempt to fill in those details here.

**Comment by pierreguillot** [comment]
I think your error is related to https://github.com/pierreguillot/Camomile/issues/81.

When you use the script `./camomile.sh` directly from the distribution folder, it generates a new folder **builds** that contains the plugins. These are the plugins that should be used. The **Camomile.so** and **CamomileLV2.so** should be used to create new plugins, not directly in your DAW.
From the documentation:
```
The Camomile plugins are a set of meta plugins. It means that the plugins 
of the distribution can't be directly loaded in a digital audio workstation but 
must be used to generate new plugins associated with Pure Data patches 
that will be loadable in the digital audio workstations.
```

**Comment by CJunit** [comment]
That is not unlikely! I compiled in the wee hours, based upon what I thought I remembered from a few months ago. I will check it this weekend.

**Comment by pierreguillot** [comment]
If the plugin looks for the file Camomile.txt, it means that the name of the plugin is Camomile but the name must not be Camomile. That's why I think you try to load directly the dynamic libraries Camomile.so or CamomileFx.so... Can you tell me how you generated the plugins? Which plugins did you try? did you tried the example? etc. 

Perhaps you could give an example of the plugin you tried to load, this way I could try to figure out the problem. And when you say that you compiled the plugin, what did you mean exactly? Why not using the pre-compiled plugins?

**Comment by CJunit** [comment]
It was all my mistake! I was tired and confused and was thinking that I compile camomile.so VSTs and add the Pd patches from within the host. I somehow missed your entire page about "generating" the plugs from the Pd patches. I used your script tonight and it worked perfectly.

Why not use the pre-compiled plugins? I did tonight, for testing. But I assuming that I can use only whatever libpd stock objects are. So my guess was that if I need any other objects, I would need to compile libpd with their source as well. Is that not the case? In any case, I will not bother you about those efforts.
